src/schema/v2/artwork/artworkContextGrids/ArtistArtworkGrid.ts

```diff
@@ -45,7 +45,7 @@ export const ArtistArtworkGridType = new GraphQLObjectType<
           options,
           artwork,
         })
-        gravityArgs.sort = "-published_at"
```

Comment:
Any tips or tricks for testing a change like this in the metaphysics tests? 

Looking like the tests mock out responses